public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -94,3 +109,60 @@ export function resolveTenantName(tenant: string, userName: string) {
     return tenant;
   }
 }
+
+function formatTenantName(tenantName: string): string {
+  if (tenantName === globalTenantName) return GLOBAL_USER_DICT.Label;
+  return tenantName;
+}
+
+export function transformRoleTenantPermissionData(
+  tenantPermissions: RoleTenantPermissionView[],
+  tenantList: Tenant[]
+): RoleTenantPermissionDetail[] {
+  return map(tenantPermissions, (tenantPermission: RoleTenantPermissionView) => {
+    const tenantNames: string[] = tenantList.map((t: Tenant) => t.tenant); // Global
+    let tenantItem = null;
+    if (
+      tenantPermission.tenant_patterns.length === 1 &&
+      tenantNames.includes(formatTenantName(tenantPermission.tenant_patterns[0]))
+    ) {
+      tenantItem = tenantList.filter((t) => {
+        return t.tenant === formatTenantName(tenantPermission.tenant_patterns[0]);
+      })[0];
+    }
```

Comment:
Add a comment here says we only consider the case that containing one tenant and for other case (multiple tenants, tenant pattern) we return N/A.

Reply:
Added comment.